evm_loader/program/src/types/mod.rs

```diff
@@ -1,5 +1,95 @@
 pub use address::Address;
+use hex::FromHex;
+use serde::de::Visitor;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use std::fmt;
+use std::ops::Deref;
 pub use transaction::Transaction;
 
 mod address;
 mod transaction;
+
+// TODO Remove dependency of neon-lib on evm-loader
+/// Wrapper structure around vector of bytes.
+#[derive(Debug, PartialEq, Eq, Default, Hash, Clone)]
+pub struct Bytes(pub Vec<u8>);
```

Comment:
This entire struct need to be removed. The sole purpose of it is to serialize `Vec<u8>` as a hex string. The same can be achieved more easily with the `hex` crate.
```
struct Foo {
    #[serde(with = "hex")]
    bar: Vec<u8>,
}
```

Reply:
I agree. I just moved the types from another module as specified in the commit message.

I would not do the change in this PR to have the fix deployed faster.

Reply:
`have the fix deployed faster` is not valid reason. It is already been in development for month. Nothing would change in a couple of days.

Reply:
Anyway, if you want to keep it for the `0x` prefix, move it to a separate file from mod.rs.
And rename it to something more appropriate. `Bytes` is already being used in a serialization context for `serde_bytes`.
I still believe it would be better to completely drop this.

Reply:
I moved and renamed the `Bytes` struct to `hexbytes::HexBytes`.

This type is also used as part of `TransactionParams` struct. This is the main reason why I do not know if removing it breaks something or not.